install-dev/controllers/http/system.php

```diff
@@ -121,9 +121,9 @@ public function display()
                         'img_dir' => $this->l('Recursive write permissions for %1$s user on %2$s', $user, '~/img/'),
                         'mails_dir' => $this->l('Recursive write permissions for %1$s user on %2$s', $user, '~/mails/'),
                         'module_dir' => $this->l('Recursive write permissions for %1$s user on %2$s', $user, '~/modules/'),
-                        'theme_lang_dir' => $this->l('Recursive write permissions for %1$s user on %2$s', $user, '~/themes/default-bootstrap/lang/'),
-                        'theme_pdf_lang_dir' => $this->l('Recursive write permissions for %1$s user on %2$s', $user, '~/themes/default-bootstrap/pdf/lang/'),
-                        'theme_cache_dir' => $this->l('Recursive write permissions for %1$s user on %2$s', $user, '~/themes/default-bootstrap/cache/'),
+                        'theme_lang_dir' => $this->l('Recursive write permissions for %1$s user on %2$s', $user, '~/themes/'._PS_DEFAULT_THEME_NAME_.'/lang/'),
+                        'theme_pdf_lang_dir' => $this->l('Recursive write permissions for %1$s user on %2$s', $user, '~/themes/'._PS_DEFAULT_THEME_NAME_.'/pdf/lang/'),
+                        'theme_cache_dir' => $this->l('Recursive write permissions for %1$s user on %2$s', $user, '~/themes/'._PS_DEFAULT_THEME_NAME_.'/cache/'),
```

Comment:
Can you replace `_PS_DEFAULT_THEME_NAME_` by `_THEME_NAME_` ? We want to avoid hardcoding the name of the default theme. This constant should be defined, see https://github.com/PrestaShop/PrestaShop/blob/develop/install-dev/init.php#L44-L47.